Automattic/woocommerce-payments
# Update minimum WC and Gutenberg version in oldest compatibility test

#### Changes proposed in this Pull Request
Changes proposed in this Pull Request 
- Update the minimum WC version in the oldest compatibility test from GH workflows
- Update the Gutenberg version in the oldest compatibility test from GH workflows
- Fix deprecation notice for Automattic\WooCommerce\Blocks\StoreApi\RoutesController
More info about the [StoreAPI](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/third-party-developers/extensibility/rest-api/extend-rest-api-add-data.md) class replacing the Package one

#### Testing instructions
Confirm all checks pass in the GH workflows

Regression test for the RoutesController update
- Make sure WooPay is enabled
- Go to the checkout page and use an email that exists in WooPay
- Enter the test OTP
- Confirm the $store_api_url is set by using RoutesController and not the fallback in the catch block. The simplest way to test it is probably with the debugger
- On WooPay, confirm the store_api_url is the same one as it was before this update

-------------------


**Post merge**
